test/core/integration/concrete/lockup/get-asset/getAsset.t.sol

```diff
@@ -3,13 +3,13 @@ pragma solidity >=0.8.22 <0.9.0;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import { Errors } from "src/core/libraries/Errors.sol";
-import { Integration_Test } from "./../../../Integration.t.sol";
-import { Lockup_Integration_Shared_Test } from "./../../../shared/lockup/Lockup.t.sol";
 
-abstract contract GetAsset_Integration_Concrete_Test is Integration_Test, Lockup_Integration_Shared_Test {
-    function setUp() public virtual override(Integration_Test, Lockup_Integration_Shared_Test) { }
+import { Integration_Test } from "../../../Integration.t.sol";
 
+abstract contract GetAsset_Integration_Concrete_Test is Integration_Test {
     function test_RevertGiven_Null() external {
+        Integration_Test.setUp();
```

Comment:
We should call this in `function setUp` and not here.

Reply:
ups, mistake, it should be completely removed